### PR TITLE
prefer delta profile image over system profile image

### PR DESCRIPTION
**Pull request by r10s**

### CODE REVIEW

**Comment by webratte** [comment]
Hi,
just a question.
Do I understand it right?

This is the way as WhatsApp handle profile pictures?

I like the way to use profile pictures I set in my local phonebook. 
And I don't like if my contacts change the profile picture because I select chats and contacts often visually via pictures I gave them.


So if I right, I would prefer if this would be optional.

Maybe somthing like:
Download remote profil pictures (set by your buddy) or use local profil pictures (set by you in the phonebook.

**Comment by r10s** [comment]
jupp, the remote user sets the profile image that is shown then. 
feel free to discuss different approaches in the forum, however, thinking over it, wrt to multi-device and multiple devices having different sets of images, i am not even sure if it is a good approach to use the system images _at all_. but again, this is more sth. to discuss in the forum :)

**Comment by webratte** [comment]
Therefore I said "optionally" 😉

Currently you use local pictures (by using the signal UI).

So I think it should be not so hard to implement a option to switch between local and remote pictures.

With an option, everybody could use it the own way.
I know many people who hate the behaviour of WhatsApp (OK they don't use WhatsApp 😂)

**Comment by hpk42** [comment]
On Fri, Nov 30, 2018 at 06:10 -0800, webratte wrote:
> Therefore I said "optionally" 😉
> 
> Currently you use local pictures (by using the signal UI).
> 
> So I think it should be not so hard to implement a option to switch between local and remote pictures.

maybe.  could you summarize how whatsapp and telegram handle this
and then do a post to the forum?
